applications/accounting/webapp/accounting/WEB-INF/web.xml

```diff
@@ -43,6 +43,11 @@ under the License.
         <param-name>partyDecoratorLocation</param-name>
         <param-value>component://accounting/widget/CommonScreens.xml</param-value>
     </context-param>
+    <context-param>
```

Comment:
Hello Pierre,
I think the same thing should be done also in ar and ap web.xml (they use screens such as invoiceOverview from accounting and will be impacted)

Reply:
HI @MkLeila,

See my reply in the ticket.